src/kedro_templar/templates.py

```diff
@@ -0,0 +1,15 @@
+from jinja2 import Environment, FileSystemLoader
```

Comment:
move to `src/kedro_templar/lib/templates.py`

Reply:
done